.github/workflows/release.yml

```diff
@@ -136,7 +138,7 @@ jobs:
 
       - name: "Upload binaries to release"
         uses: svenstaro/upload-release-action@v2
-        if: github.event_name == 'release'
+        if: github.event_name == 'release' || github.event_name == 'schedule'
```

Comment:
LGTM but I'd like to understand what it'll actually look like on the release page before you merge this in (title, description, etc.).